### PR TITLE
Add more instructions for Windows.

### PR DESCRIPTION
**Pull request by floitsch**
The added instructions made it possible to get compiling on a virtual-box image provided by Microsoft.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,28 @@ We're actively working on simple installation steps for other platforms. For now
 
 ### Build system
 
+#### Windows
```
Comment (kasperl) [inline README.md:100]: I'd probably put Windows below Linux and Mac here and elsewhere.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: README.md (hunk 1 above)
Comment (floitsch) [reply README.md]: done.

File: README.md [hunk 2]
```diff
@@ -244,7 +273,17 @@ There are syntax highlighters for VIM and CodeMirror in the
 ## Build for ESP32
 
 Make sure the environment variables for the ESP32 tools are set, as
-described in the [dependencies](#dependencies) section.
+described in the [dependencies](#dependencies) section. Typically this
+consists of running the following command:
+
+``` sh
+# On Windows:
+third_party/esp-idf/install.bat
```
[the remark this answers is not in the document]
Comment (floitsch) [reply README.md]: done. (needed to change the 'sh' to `powershell`.

**Pull request merged**